guides/common/modules/proc_using-tftp-services-through-nat.adoc

```diff
@@ -11,11 +11,12 @@ endif::[]
 ifndef::satellite[]
 .Using NAT on Linux with `firewalld`:
 endif::[]
-* Use the following command to allow TFTP service on UDP port 69, load the kernel TFTP state tracking module, and make the changes persistent:
+. Allow TFTP service on UDP port 69 and load the kernel TFTP state tracking module:
```

Comment:
```suggestion
. Allow the TFTP service in the firewall configuration:
```
Non-blocking, but I think it contains a lot of unnecessary information.

Reply:
Applied.